.github/workflows/chromatic.yml

```
@@ -0,0 +1,23 @@
+name: "Chromatic"
+
+on:
+    pull_request:
+        types: [ labeled ]
+
+jobs:
+  chromatic:
+    if: ${{ github.event.label.name == 'chromatic' }}
```

If we add the synchronize pull request type, we also need to check for the label name on the pull-request itself.

```suggestion
    if: ${{ github.event.label.name == 'chromatic' || contains(github.event.pull_request.labels.*.name, 'chromatic') }}
```